Xamarin.Forms.Core/DesignMode.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms.Core
+{
+    public static class DesignMode
+    {
+		/// <summary>
```

Comment:
we don't do inline doc. our doc tool can't even extract it from there

Reply:
What's the proper way? I added that for intellisense.